README.md

```diff
@@ -149,6 +149,18 @@ And when you are ready to migrate to native USDC, run:
 yarn script:migrate:broadcast
 ```
 
+### What will circle need at migration?
+
+#### 1. Circle will need the metadata from the original deployment of the USDC implementation that was used
+  
+To do this you will need to go back to the `stablecoin-evm` github repo that the implementation was deployed from inorder to extract the raw metadata from the compiled files you can run a command like this
+
+```bash
```

Comment:
This was on the `artifacts/` folder when I run it. Could we add a comment with that? That might not be `out/` but rather `artifacts/` or any similar folder?